src/input/ExpressLRSFiveWay.cpp

```diff
@@ -101,7 +102,7 @@ void ExpressLRSFiveWay::update(int *keyValue, bool *keyLongPressed)
         }
         // else still pressing, waiting for long if not already signaled
         else if (!isLongPressed) {
-            if ((now - keyDownStart) > KEY_LONG_PRESS_MS) {
+            if (!Throttle::isWithinTimespanMs(keyDownStart, KEY_LONG_PRESS_MS)) {
                 *keyValue = keyInProcess;
                 *keyLongPressed = true;
                 isLongPressed = true;
```

Comment:
There are some calls below in nt32_t ExpressLRSFiveWay::runOnce() that might be worth looking into too...